src/calculator/pump/PumpData.cpp

```diff
@@ -0,0 +1,414 @@
+//
+// Created by zf9 on 7/16/18.
+//
+
+#include "calculator/pump/PumpData.h"
+
+PumpData::PumpData(std::string manufacturer, std::string model, std::string type, std::string serialNumber,
+                   std::string status,  std::string pumpType, std::string radialBearingType, std::string thrustBearingType,
+                   std::string shaftOrientation, std::string shaftSealType, std::string fluidType, std::string priority,
+                   std::string driveType, std::string flangeConnectionClass, std::string flangeConnectionSize,
+                   int numShafts, int speed, int numStages,  int yearlyOperatingHours, int yearInstalled,
+                   int finalMotorRpm, double inletDiameter, double weight, double outletDiameter,
+                   double percentageOfSchedule, double dailyPumpCapacity, double measuredPumpCapacity,
+                   double pumpPerformance, double staticSuctionHead, double staticDischargeHead, double fluidDensity,
+                   double lengthOfDischargePipe, double pipeDesignFrictionLosses, double maxWorkingPressure,
+                   double maxAmbientTemperature, double maxSuctionLift, double displacement, double startingTorque,
+                   double ratedSpeed, double shaftDiameter, double impellerDiameter, double efficiency,
+                   double output60Hz, double minFlowSize, double pumpSize, bool outOfService)
+        : manufacturer(std::move(manufacturer)), model(std::move(model)), type(std::move(type)), serialNumber(std::move(serialNumber)),
```

Comment:
@FontenotZach Oh I see what you are doing now. You are passing by copy then `std::move`ing it in the initialization list. Ok, I am ambivalent about this approach versus passing by `const std::string &` then the initialization list copying into the member variable.